### PR TITLE
DrivAerML: 4L/256d+T_max=30 — lr=8e-4 (higher LR)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The current DrivAerML baseline uses lr=5e-4. Testing a higher learning rate of 8e-4 may allow faster convergence on the large DrivAerML dataset during early epochs, potentially reaching a better minimum within the epoch budget. Higher LRs with cosine annealing can escape shallow local minima and explore more of the loss landscape before annealing to a precise final value. This is a bracketing experiment alongside a lower-LR run (lr=3e-4) to find the optimal LR for this architecture.

## Instructions
Train the DrivAerML model with the baseline 4L/256d configuration but increase the learning rate from 5e-4 to 8e-4:

```bash
cd target/icml2026 && python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 8e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --model-layers 4 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group drivaerml-4L256d-lr
```

**CRITICAL flags — do NOT omit these:**
- `--batch-size 1`
- `--drivaerml-train-surface-points 50000`
- `--drivaerml-eval-surface-points 50000`
- `--max-train-batches 394`
- `--max-eval-batches 200`
- `--model-heads 4`
- `--no-use-ema`
- `--enable-fourier`
- `--wandb_group drivaerml-4L256d-lr` (groups this run with the lr=3e-4 run for comparison)

The only change from baseline is `--lr 8e-4` (was `5e-4`).

## Baseline
Current best DrivAerML metrics (PR #2550):
- **val_primary/surface_rel_l2_pct**: 12.96% (lower is better)
- **test_primary/surface_rel_l2_pct**: 14.41%
- Baseline W&B run: 8s5i8y06
- Baseline reproduce command:
  ```
  cd target/icml2026 && python train.py --dataset drivaerml --optimizer adamw --lr 5e-4 --cosine_t_max 30 --no-use-ema --enable-fourier --model-layers 4 --model-hidden-dim 256 --model-heads 4 --epochs 999 --batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200
  ```
- External target: <3.71% (AB-UPT, ~500 epochs) — 3.5x gap remaining

**Report val_primary/surface_rel_l2_pct in your PR results.**

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2600 because ADVISOR: Closing — DrivAerML baseline is 5.73% (4L/384d, PR #2602). All 4L/256d experiments are obsolete. You'll receive a new 4L/384d assignment shortly..
